### ISSUE TITLE
Password forget what is after a space

### ISSUE THREAD

**Issue by supermamie** [YunoHost-Apps/borg_ynh#49]
On initialisation, I set my passphrase to
> Save of my server !

My last working save is on july 13
Since that, I have errors about passphrase
I tried
> sudo yunohost app setting borg passphrase

And it only return
> Save

I tried to restore it
> sudo yunohost app setting borg passphrase -v "Save of my server !"

But the save still fails.
I tried a command suggested on the forum to get the actually used password
> sudo grep "BORG_PASSPHRASE=" /etc/yunohost/hooks.d/backup_method/05-borg_app | cut -d'"' -f2

And it still returns
> Save

So I edited the file, and the code only contains `BORG_PASSPHRASE="Save"`
I replaced it with the real password and now it works.

I don't know how, but my password was truncated.

There were a commit on July 14 containing
```
export passphrase=$(ynh_app_setting_get $app passphrase)
```
Maybe this needs some escaping to save the spaces ?

**Comment by SimonLefort** [comment]
Hello, 
I had a similar problem and I dug.

The problem is, I think, in this script : 
```
# cat /etc/yunohost/hooks.d/backup_method/05-borg_app
(...)
BORG_PASSPHRASE="RGjauGbbg6d$jj77g*piabfbfGedE"
(...)
```
(I changed the passphrase, obviously..)

I chose a strong passphrase and I had "$jj77g" and this is considerate like an empty variable. Finally, borg used "RGjauGbbg6d*piabfbfGedE". We have to change " -> '.

```
$ PASSPHRASE="gaghtbgg$TEXT*gloa"
$ echo $PASSPHRASE 
gaghtbgg*gloa

$ PASSPHRASE='gaghtbgg$TEXT*gloa'
$ echo $PASSPHRASE 
gaghtbgg$TEXT*gloa
```

Perhaps we should add a warning during installation to say it's better to avoid some special characters ( ' , " , $ , .. ) ?

**Comment by zamentur** [comment]
Fixed in #71
